deployer/tasks/setup.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Deployer;
+
+use Deployer\Task\Context;
+use Symfony\Component\Yaml\Yaml;
+
+desc('Setup redaxo instance');
+task('setup', new class {
```

Comment:
warum nicht einfach ein closure?

Reply:
Kommt ja noch mehr, und dachte mir, das bisschen zu gruppieren in einzelne Methoden.
Aber vielleicht ändere ich das auch noch wieder.